helpers/deploymentConfig.ts

```diff
@@ -407,130 +447,154 @@ export const bridgeConfig: BridgeConfig = {
+      ...createMethodEntries(
+        parseInt(preconfiguredAddresses.unichainmainnet.LzVirtualChainId),
+        300000,
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+      ),
     ],
   },
   opmainnet: {
     methods: [
       ...createMethodEntries(
-        102,
+        parseInt(preconfiguredAddresses.bscmainnet.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        101,
+        parseInt(preconfiguredAddresses.ethereum.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        202,
+        parseInt(preconfiguredAddresses.opbnbmainnet.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        110,
+        parseInt(preconfiguredAddresses.arbitrumone.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
       ...createMethodEntries(
-        165,
+        parseInt(preconfiguredAddresses.zksyncmainnet.LzVirtualChainId),
         300000,
         parseUnits("100000", 18),
         parseUnits("20000", 18),
         parseUnits("102000", 18),
         parseUnits("20400", 18),
       ),
-      ...createMethodEntries(184),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.basemainnet.LzVirtualChainId)),
+      ...createMethodEntries(
+        parseInt(preconfiguredAddresses.unichainmainnet.LzVirtualChainId),
+        300000,
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+        parseUnits("0", 18),
+      ),
     ],
   },
   basesepolia: {
     methods: [
-      ...createMethodEntries(10102),
-      ...createMethodEntries(10202),
-      ...createMethodEntries(10161),
-      ...createMethodEntries(10248),
-      ...createMethodEntries(10232),
-      ...createMethodEntries(10231),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.bsctestnet.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.opbnbtestnet.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.sepolia.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.arbitrumsepolia.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.zksyncsepolia.LzVirtualChainId)),
+      ...createMethodEntries(parseInt(preconfiguredAddresses.opsepolia.LzVirtualChainId)),
```

Comment:
I miss the entry for `unichainsepolia`:
```
...createMethodEntries(parseInt(preconfiguredAddresses.unichainsepolia.LzVirtualChainId)),
```

Reply:
https://github.com/VenusProtocol/token-bridge/pull/49/commits/6d39c6357cec3cb1b89a9417aadc458875b50841